data/strings/types_strings.txt

```diff
@@ -2219,7 +2223,7 @@
     ja = 神社
     ko = 성지
     nb = Alter
-    pl = Kapliczka
+    pl = Kapliczka szintoistyczna 
```

Comment:
Please remove the trailing space.

Reply:
Yes, there is a unnecessary space, I'll fix it, thanks for info

Reply:
@mpimenov Have no idea how to edit my commit :-( Could you assist?

Reply:
It's ok if you make a separate commit with the changes.

Reply:
> It's ok if you make a separate commit with the changes.

OK, done - all fixes! And I've added some other localized strings. Already should be fine.